api/types/types.go

```diff
@@ -528,3 +528,8 @@ type PushResult struct {
 	Digest string
 	Size   int
 }
+
+// BuildResult contains the image id of a successful build
+type BuildResult struct {
```

Comment:
Not sure I follow, the client and the server both use it so they have a mutual understanding of the JSON content of the Aux message. I think it's the same as the `PushResult` struct just above in that sense.